lib/plugins/package/lib/zipService.js

```diff
@@ -194,7 +194,7 @@ function excludeNodeDevDependencies(servicePath) {
             return fs
               .readFileAsync(depFile)
               .then((fileContent) =>
-                _.uniq(fileContent.toString('utf8').split(os.EOL)).filter(Boolean)
+                _.uniq(fileContent.toString('utf8').split(/\r\n|\n/)).filter(Boolean)
```

Comment:
I think we can do something as `[\r\n]+` (?)

Reply:
I guess we can as `+` will be greedy so it should split everything properly :+1: